web/templates/empty_list_widget.hbs

```diff
@@ -0,0 +1,11 @@
+{{#if is_container_table}}
+    <tr>
```

Comment:
It seems like we should just have two separate templates for the table version and the list version, and then just push up the conditional into actual JavaScript.  The two code paths don't seem to have enough in common to justify having all these ugly conditionals in the template.

Reply:
I've updated the code as suggested.

Is it okay to have templates with just one line of code? Having a separate template for a single line seems odd to me.